fonts/squarize.py

```diff
@@ -210,144 +243,164 @@ def initialize_glyphs():
         newfont.selection.select(("singletons",), unislots[i])
         newfont.paste()
 
-#function in which we initialize the lenghts, depending on the font
 def initialize_lengths():
-    global base_height, line_width, width_punctum, width1, width2, width_debilis, width_deminutus, width_inclinatum_deminutus, width_flexusdeminutus, porrectusflexuswidths, porrectuswidths, width_inclinatum, width_stropha, hepisemus_additional_width, width_oriscus, width_quilisma, width_high_pes, width_oriscus_rev
-    if (font_name=="gregorio"):
+    "Initialize the lenghts, depending on the font"
+    global BASE_HEIGHT, LINE_WIDTH, WIDTH_PUNCTUM, width1, width2
+    global WIDTH_DEBILIS, WIDTH_DEMINUTUS, WIDTH_INCLINATUM_DEMINUTUS
+    global WIDTH_FLEXUSDEMINUTUS, PORRECTUSFLEXUSWIDTHS, PORRECTUSWIDTHS
+    global WIDTH_INCLINATUM, WIDTH_STROPHA, HEPISEMUS_ADDITIONAL_WIDTH
+    global WIDTH_ORISCUS, WIDTH_QUILISMA, WIDTH_HIGH_PES, WIDTH_ORISCUS_REV
+    if (font_name == "gregorio"):
         # the base heigth is half the space between two lines plus half the heigth of a line
-        base_height=157.5
+        BASE_HEIGHT = 157.5
```

Comment:
BASE_HEIGHT could be a global variable, Gregorio won't accept a different value anyway...